Gems/ROS2/Code/Source/Camera/ROS2CameraSensorComponent.h

```diff
@@ -22,6 +22,15 @@
 
 namespace ROS2
 {
+    namespace CameraConstants
+    {
+        const char* const ImageMessageType = "sensor_msgs::msg::Image";
+        const char* const DepthImageConfig = "Depth Image";
+        const char* const ColorImageConfig = "Color Image";
+        const char* const InfoConfig = "Camera Info";
+        const char* const CameraInfoMessageType = "sensor_msgs::msg::CameraInfo";
```

Comment:
Since these are in the header they'd need `inline`. I would also put `constexpr` as well.

Reply:
Thanks for the suggestion, adjusted it to inline constexpr.